src/nodejs/neon-js/cli/rpc/getBlockCount.js

```diff
@@ -8,30 +8,60 @@ const program = require('commander')
 const _       = require('underscore')
 
 const neon      = require('@cityofzion/neon-js')
-const dbg     = require('nodejs_util/debug')
+const dbg       = require('nodejs_util/debug')
+const netutil   = require('nodejs_util/network')
+
+var cfg       = require('nodejs_config/config.js')
+var config    = cfg.load('nodejs_config/nodejs.config.json')
+
+let nodes  = []
+let defly = false
 
 function print(msg) {
   console.log(msg);
 }
 
 program
-  .version('0.1.0')
-  .usage('-n <node>')
+  .version('0.2.0')
+  .usage('')
   .option('-d, --debug', 'Debug')
-  .option('-n, --node <node>', 'set RPC node to use (be sure to preface with https://)')
+  .option('-n, --node [node]', 'set RPC node to use (be sure to preface with https://), if not provided will try to use node with tallest block')
+  .option('-N, --Net [Net]', 'Select network [net]: i.e., TestNet or MainNet', 'TestNet')
```

Comment:
why `Net` with capital N here?

Reply:
So it doesn't get confused with -n node argument.